.ci/docker/buildtools-windows/Dockerfile

```diff
@@ -6,21 +6,26 @@ FROM mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-1909
 # Restore the default Windows shell for correct batch processing.
 SHELL ["cmd", "/S", "/C"]
 
-# Download the Build Tools bootstrapper.
-ADD https://aka.ms/vs/16/release/vs_buildtools.exe C:\TEMP\vs_buildtools.exe
-
-# Install Build Tools excluding workloads and components with known issues.
-RUN C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache `
-    --installPath C:\BuildTools `
-    --all `
-    --remove Microsoft.VisualStudio.Component.Windows10SDK.10240 `
-    --remove Microsoft.VisualStudio.Component.Windows10SDK.10586 `
-    --remove Microsoft.VisualStudio.Component.Windows10SDK.14393 `
-    --remove Microsoft.VisualStudio.Component.Windows81SDK `
- || IF "%ERRORLEVEL%"=="3010" EXIT 0
-
-# Start developer command prompt with any other commands specified.
-ENTRYPOINT C:\BuildTools\Common7\Tools\VsDevCmd.bat &&
+# Download NuGet v5.4.0 
+ADD https://dist.nuget.org/win-x86-commandline/v5.4.0/nuget.exe C:\nuget\nuget.exe
+
+# Add NuGet to Beginning of Path (We want this version of Nuget before any other versions that may be in the path)
+RUN @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "[Environment]::SetEnvironmentVariable(\"Path\", \"C:\nuget;\" + $env:Path, [EnvironmentVariableTarget]::Machine)"
```

Comment:
Could be this powershell command simplify if https://github.com/elastic/apm-agent-dotnet/pull/689/files#diff-43c7fcefff21421ffd113b8ffbee9977R7 gets changed to use the powershell shell?

I guess this particular suggestion could be only interested if moving forward with the chocolatey installation.

Reply:
Would prefer to use this version of the script while it seems complex it is pretty straightforward and doesn't require us to add a ps1 file to execute to perform this action.

Reply:
I don't mean to move those commands to another script and call it from this Dockerfile.

I mean something like the below snippet

```
## Setup the PS1 shell
SHELL ["C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe", "-command"]
...

## Then all the ps commands in the RUN step could be simplified
```

Anyway, this is not outdated.